python/paddle/tensor/math.py

```diff
@@ -1622,3 +1624,80 @@ def cumsum(x, axis=None, dtype=None, name=None):
             kwargs[name] = val
     _cum_sum_ = generate_layer_fn('cumsum')
     return _cum_sum_(**kwargs)
+
+def prod(x, axis=None, keepdim=False, dtype=None, name=None):
+    """
+    Compute the product of tensor elements over the given axis.
+
+    Args:
+        x(Tensor): Input of prod operator. The data type is float32, float64, int32, int64.
```

Comment:
The data type is float32, float64, int32, **or** int64

Reply:
Thanks, done.